src/commands/init.command.ts

```diff
@@ -60,3 +51,41 @@ const promptForDbUrl = async (): Promise<string> => {
   ]);
   return dbUrl;
 };
+
+const copyTemplate = async ({
+  templatePath,
+  workspacePath,
+  secrets: { dbUrl },
+}: {
+  templatePath: string;
+  workspacePath: string;
+  secrets: { dbUrl: string };
+}) => {
+  await fs.copy(templatePath, workspacePath, { overwrite: true });
+  const envPath = path.join(workspacePath, 'apps/server/.env.local');
+  await fs.ensureFile(envPath);
+  await fs.appendFile(envPath, `\nMONGODB_URI='${dbUrl}'`);
+};
+
+const installDependenciesWithMessage = async (workspacePath: string) => {
+  const spinner = ora('Installing dependencies').start();
+  await installDependencies({
+    cwd: workspacePath,
+    silent: true,
+  });
+  spinner.succeed('Installed dependencies');
+};
+
+const startApplications = async (workspacePath: string) => {
+  console.log(`${logSymbols.info} Starting applications...`);
```

Comment:
This outputs the final line in the screenshot below
<img width="350" alt="image" src="https://github.com/mantis-apps/mantis-cli/assets/20916810/bbe0f4bf-e7b2-4f5b-b2de-b3ad7f64f515">
I considered using the `OrbitLogger`, but noticed it visually clashed with the style of the other console tools we use
<img width="488" alt="image" src="https://github.com/mantis-apps/mantis-cli/assets/20916810/022af3d2-8e04-47b5-a234-2712b3122c96">

I'm not opposed to using the `OrbitLogger`, but made this change in the meantime in case visual consistency was more important. If we prefer the output of the `OrbitLogger`, I can add it back in.